pandas/core/arrays/timedeltas.py

```diff
@@ -143,6 +144,23 @@ def _box_func(self, x: np.timedelta64) -> Timedelta | NaTType:
             return NaT
         return Timedelta._from_value_and_reso(y, reso=self._reso)
 
+    # error: Decorated property not supported
+    @property  # type: ignore[misc]
+    @doc(dtl.DatetimeLikeArrayMixin.freq)
+    def freq(self) -> Tick:
```

Comment:
TDA.freq _can_ be None.  its only PeriodArray.freq that can't be None

Reply:
Changed :)